packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page/entity-editor/links-section/link-table/cells/linked-with-cell/linked-with-cell-editor.tsx

```diff
@@ -0,0 +1,170 @@
+import { ProvideEditorComponent } from "@glideapps/glide-data-grid";
+import { Entity } from "@hashintel/hash-subgraph";
+import { getRoots } from "@hashintel/hash-subgraph/src/stdlib/roots";
+import { useRouter } from "next/router";
+import { useEffect, useRef, useState } from "react";
+import { useBlockProtocolAggregateEntities } from "../../../../../../../../../components/hooks/blockProtocolFunctions/knowledge/useBlockProtocolAggregateEntities";
+import { useBlockProtocolArchiveEntity } from "../../../../../../../../../components/hooks/blockProtocolFunctions/knowledge/useBlockProtocolArchiveEntity";
+import { useBlockProtocolCreateEntity } from "../../../../../../../../../components/hooks/blockProtocolFunctions/knowledge/useBlockProtocolCreateEntity";
+import { generateEntityLabel } from "../../../../../../../../../lib/entities";
+import { HashSelectorAutocomplete } from "../../../../../../../types/entity-type/hash-selector-autocomplete";
+import { useEntityEditor } from "../../../../entity-editor-context";
+import { LinkedWithCell } from "../linked-with-cell";
+
+export const LinkedWithCellEditor: ProvideEditorComponent<LinkedWithCell> = (
+  props,
+) => {
+  const { entitySubgraph, refetch } = useEntityEditor();
+  const { createEntity } = useBlockProtocolCreateEntity();
+  const { archiveEntity } = useBlockProtocolArchiveEntity();
+  const { aggregateEntities } = useBlockProtocolAggregateEntities();
+
+  const { value: cell, onFinishedEditing } = props;
+  const { expectedEntityTypes, linkAndTargetEntities, linkEntityTypeId } =
+    cell.data.linkRow;
+
+  const [search, setSearch] = useState("");
+
+  const [entities, setEntities] = useState<Entity[]>([]);
+  const [loading, setLoading] = useState(true);
+  const [open, setOpen] = useState(false);
+  const highlightedRef = useRef<null | Entity>(null);
+  const router = useRouter();
+
+  useEffect(() => {
+    const init = async () => {
+      try {
+        setLoading(true);
+        const { data } = await aggregateEntities({
+          data: {
+            rootEntityTypeIds: expectedEntityTypes.map(
+              ({ schema }) => schema.$id,
+            ),
+          },
+        });
+
+        if (data) {
+          setEntities(getRoots(data));
+        }
+      } finally {
+        setLoading(false);
+      }
+    };
+
+    void init();
+  }, [aggregateEntities, expectedEntityTypes]);
+
+  const onCreateNew = () => {
+    if (!expectedEntityTypes[0]) {
+      return;
+    }
+
+    const accountSlug = router.query["account-slug"];
+    /** @todo ask about this behavior */
+    void window.open(
+      `/${accountSlug}/entities/new?entity-type-id=${encodeURIComponent(
```

Comment:
Think this URL is now `@shortname/new/entity`

Reply:
Correct, updated the URL